lib/themes/dosomething/paraneue_dosomething/scss/content/_taxonomy.scss

```diff
@@ -1,17 +1,18 @@
 .taxonomy-term {
-  .container__title,
-  .sources {
-    @include compacted();
-  }
-
-  .container__body,
-  .additional-text {
-    p,
-    ul,
-    ol {
-      @include compacted();
-    }
-  }
+//  .container__title,
+//  .sources {
+//    @include compacted();
+//  }
+//
+//  .container__body,
+//  .additional-text {
+//    p,
+//    ul,
+//    ol {
+//      @include compacted();
+//    }
+//  }
+  // @TODO ^ What was this?
```

Comment:
The compacted class was added to each individual element to make it visually narrower, but I think this was just a temp fix...?